### PR TITLE
Feature load product

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
added all functionalities on product listing page except sideNav

### CODE REVIEW

File: src/pages/Cart/Cart.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.cart-summary-heading {
+  display: none;
+}
+
+.summary-items {
+  display: flex;
+  justify-content: space-between;
+  color: #000;
```
Comment (Rohanmond) [inline src/pages/Cart/Cart.css:163]: better to use color variables

**Review by Rohanmond — APPROVED** [review]
PR is looking good to merge

File: src/components/ProductCard/Productcard.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
+.product-card {
+  height: 400px;
+  display: flex;
+  flex-flow: column wrap;
+  background-color: #fff;
```
Comment (Shah-Arsalan) [inline src/components/ProductCard/Productcard.css:5]: use color variables

File: src/components/ProductCard/Productcard.jsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      }
+
+      if (res.status === 200 || res.status === 201) {
+        dispatch({
+          type: actionTypes.SET_WISHLIST,
+          payload: { wishlist: res.data.wishlist },
+        });
+      }
+    } catch (err) {
+      console.log(err);
```
Comment (Shah-Arsalan) [inline src/components/ProductCard/Productcard.jsx:37]: You can also go with console.error . Just pointing it out  . Nothing too important

**Review by Shah-Arsalan — APPROVED** [review]
You can merge the pull request

**Pull request merged**